sefaria/sheets.py

```diff
@@ -874,6 +889,9 @@ def get_sheets_for_ref(tref, uid=None, in_collection=None):
 				"category":        "Sheets", # ditto
 				"type":            "sheet", # ditto
 			}
+			if include_collections:
+				sheet_data["collections"] = sheet_id_to_collection[sheet["id"]]
+				sheet_data["dateCreated"] = sheet["dateCreated"]
```

Comment:
why is dateCreated part of this if statement?

Reply:
I wasn't sure where to put it.  It's supposed to show up next to the user profile in SearchSheetResult now.  It's not necessary for the related API to have access to dateCreated, so I didn't add dateCreated to the big "sheet_data = {...}" but it is necessary for the sheets ref API to have access to dateCreated.  But it is definitely confusing to set dateCreated only when include_collections is True, so maybe dateCreated should just be set in the original "sheet_data = {...}".  What do you think?